easybuild/toolchains/mpi/mpich2.py

```diff
@@ -45,9 +45,10 @@ class Mpich2(Mpich):
     def _set_mpi_compiler_variables(self):
         """Set the MPICH_{CC, CXX, F77, F90, FC} variables."""
 
-        # hardwire MPI wrapper commands (otherwise Mpich parent class sets them based on MPICH version)
-        self.MPI_COMPILER_MPIF77 = 'mpif77'
-        self.MPI_COMPILER_MPIF90 = 'mpif90'
-        self.MPI_COMPILER_MPIFC = 'mpif90'
+        if self.MPI_COMPILER_MPIF77 is None and self.MPI_COMPILER_MPIF90 is None and self.MPI_COMPILER_MPIFC is None:
+            # hardwire MPI wrapper commands (otherwise Mpich parent class sets them based on MPICH version)
```

Comment:
OK, I guess clarifying wouldn't hurt.

@akesandgren can you include a comment that only the Fortran compiler commands need to be overridden, i.e. that the C/C++ compilers are correctly set via the parent class?